source/dagger/gameplay/plight/topdown_controller.cpp

```diff
@@ -29,10 +36,9 @@ void TopdownControllerSystem::SpinUp()
 
 void TopdownControllerSystem::Run()
 {
-    Engine::Registry().view<InputReceiver, Sprite,Transform, Animator, TopdownCharacter, CombatStats>().each(
-        [](const InputReceiver input_, Sprite& sprite_,Transform& transform_, Animator& animator_,TopdownCharacter& char_, CombatStats& cstats_)
+    Engine::Registry().view<InputReceiver, Sprite,Transform, Animator, TopdownCharacter, CombatStats , PlightCrosshair>().each(
+        [](const InputReceiver input_, Sprite& sprite_,Transform& transform_, Animator& animator_,TopdownCharacter& char_, CombatStats& cstats_, PlightCrosshair& crosshair_)
         {
-            Float32 moveX = input_.values.at("moveX");
             Float32 moveY = input_.values.at("moveY");
```

Comment:
might be better to call it moveForward everywhere

Reply:
True , it doesn't really make sense to call it moveY anymore . Will change it :)

Reply:
Changed it to just move , since we can also go backwards :D